pytorch_msssim/ssim.py

```diff
@@ -54,19 +54,25 @@ def gaussian_filter(input, win):
     return out
 
 
-def _ssim(X, Y, data_range, win, size_average=True, K=(0.01, 0.03)):
-
+def _ssim(
+    X: Tensor,
+    Y: Tensor,
+    data_range: float,
+    win: Tensor,
+    size_average: bool = True,
+    K: Union[Tuple[float, float], List[float]] = (0.01, 0.03)
```

Comment:
I think the type of K should be `Tuple[float, float]` rather than `Union[Tuple[float, float], List[float]]`.
for user using pylance or others linter, we should infer this parameter to be a `Tuple[float, float]`, though list of two float also works well on unpacking.